hoprd/hoprd/example_cfg.yaml

```diff
@@ -160,10 +154,23 @@ hopr:
         # If set, the strategy will redeem only aggregated tickets.
         redeem_only_aggregated: true
 
+        # The strategy will automatically redeem if there's a single ticket left when a channel
+        # transitions to `PendingToClose` and it has at least this value of HOPR.
+        # This happens regardless the `redeem_only_aggregated` setting.
+        on_close_redeem_single_tickets_value_min: "2000000000000000000 HOPR"
+
       # Passive strategy does nothing. This is equivalent as leaving
       # the `strategies` array empty.
       #- !Passive
 
+      # Strategy that monitors channels that are in `PendingToClose` state and
+      # their channel closure grace period has already elapsed, and on more issuing
+      # channel close transaction on these channels to finalize the closure.
+      # - !ClosureFinalizer
```

Comment:
Should it not be something like "OnChannelClosureRedeeming`?

Reply:
I really find it hard to make up a good name for this 😁